### DISCUSSION TITLE
Vita - update version 1.5.1 error

### DISCUSSION THREAD

**Discussion by kadoon** [diasurgical/DevilutionX#6717] [Q&A] [answered]
Hi,

I have version 1.4.0 installed on my vita

I try to update (only execute .vpk, not change the actual archives)

Update 1.4.0 to 1.4.1 = OK
Update 1.4.1 to 1.5.0 = OK
Update 1.5.0 to 1.5.1 = ERROR: Error Failed to load UI resources. Make sure devilutionx.mpq is in the game folder and that it is up to date.

->The error occurs when opening devilutionX. The vpk file execution goes smoothly

Should I do something more? the file in question exists in the correct folder.

follow SDL.log:
[SDL_Log.txt](https://github.com/diasurgical/devilutionX/files/12923119/SDL_Log.txt)

**Comment by Trihedraf** [comment, marked as answer]
Delete the devilutionx.mpq as it is no longer needed for the Vita version in 1.5.1

**Reply by kadoon** [reply]
This work. Thanks
